### ISSUE TITLE
Licensing for custom KASM images

### ISSUE THREAD

**Issue by j-simmons-phd** [kasmtech/workspaces-core-images#7]
I am working on some custom KASM images based on the [user documentation](https://kasmweb.com/docs/latest/how_to/building_images.html).  I plan to publish these images on GitHub.  I found the [license file](https://kasmweb.com/support/policies/on-premise_end_user_license_agreement.pdf), however, it is not clear to me how I can license my own custom images after reading the license file given the need to include the code from the user documentation in my docker files.

Can you clarify which licenses I can use for my own custom images and remain compliant with your licensing?

Thank you.

**Comment by mmcclaskey** [comment]
Hello, thank you for pointing that out, I've created an internal ticket to ensure we address the lack of licensing on some of our open source projects. It will take us just a bit of time, because we have to evaluate the use of any open source software and take their license into account. KasmVNC is used within this project, for example, and that has a GLP-2.0 license.

**Comment by j-simmons-phd** [comment]
You are most welcome.  And I understand the need to take the time to review the other relevant licenses involved.  I'll keep an eye on this issue for updates.  Thanks!

**Comment by j-simmons-phd** [comment]
Bumping this thread.  Any news?

**Comment by capsulecorplab** [comment]
I've also been working on a few custom KASM based workspace images and would appreciate guidelines on licensing.

**Comment by TristanDamron** [comment]
Ditto! I'm also building a custom KASM-based image and would like some guidance here.

**Comment by mmcclaskey** [comment]
Hello all. A LICENSE.md file has been added to the Workspaces Core and Workspaces projects respectively. Bottom line, we applied an MIT open source license to both projects with a disclaimer (that likely goes without saying) that the license only applies to source code directly within the project and does not cover any dependencies (3rd party or Kasm) that are referenced but not maintained in the respective Git repo. Sorry for the delay on this.
